src/server.ts

```diff
@@ -102,19 +102,22 @@ const apolloServer = new ApolloServer({
     role: RoleAuthorizationDirective,
   },
   context: ({ req, res, connection }) => {
+    const apiRootUrl = `${req.protocol}://${req.get("host")}/`;
```

Comment:
Add a detailed comment here why this is needed.

Reply:
@xoldyckk Done